### PR TITLE
[ONEAPP-16472] Zigbee Multi Switch: Orvibo switch not reporting correct status on join

### PR DESCRIPTION
**Pull request by FenMei007S**

### CODE REVIEW

**Comment by FenMei007S** [comment]
@tpmanley plz review it

**Review by tpmanley — APPROVED** [review]

**Comment by tpmanley** [comment]
I'll merge once the merge window is open again

**Pull request merged**
